src/main.rs

```diff
@@ -403,9 +411,29 @@ mod tests {
                 .trim()
                 .to_string()
         }
+        fn get_pci_device_ids(&self) -> String {
+            self.ssh_command("cat /sys/bus/pci/devices/*/device")
+                .trim()
+                .to_string()
+        }
+
+        fn get_pci_vendor_ids(&self) -> String {
+            self.ssh_command("cat /sys/bus/pci/devices/*/vendor")
+                .trim()
+                .to_string()
+        }
+
+        fn check_console_status(&self) -> bool {
```

Comment:
Nit: I would call this function `console_detected` or `is_console_detected`.

Reply:
Agreed.

Reply:
Done!